ivy/functional/ivy/manipulation.py

```diff
@@ -365,15 +370,67 @@ def reshape(
     ret
         Reshaped array.
 
-    Examples
-    --------
+    Functional Examples
+    -------------------
+    With :code:`ivy.Array` input:
+
     >>> x = ivy.array([[1,2,3], [4,5,6]])
     >>> y = ivy.reshape(x, (3,2))
     >>> print(y)
     ivy.array([[1, 2],
                [3, 4],
                [5, 6]])
 
+    With :code:`ivy.NativeArray` input:
+
+    >>> x = ivy.native_array([[0, 1, 2, 3]])
+    >>> y = ivy.reshape(x, (2, 2))
+    >>> print(y)
+    ivy.array([[0, 1],
+               [2, 3]])
+
+    With :code:`ivy.Container` input:
+
+    >>> x = ivy.Container(a=ivy.array([[0., 1., 2.]]), b=ivy.array([[3., 4., 5.]]))
+    >>> y = ivy.reshape(x, (-1, 1))
+    >>> print(y)
+    {
+        a: ivy.array([[0.],
+                      [1.],
+                      [2.]]),
+        b: ivy.array([[3.],
+                      [4.],
+                      [5.]])
+    }
+
+
+    Instance Method Examples
+    ------------------------
+
+    Using :code:`ivy.Array` instance method:
+
+    >>> x = ivy.array([[0., 1., 2.]])
+    >>> y = x.reshape(3, 1)
```

Comment:
on line number 413 of functional/ivy/manipulation.py, the argument should be a tuple.